intro.js

```diff
@@ -262,7 +262,11 @@
       }
       var temp = {};
       for (var key in object) {
-          temp[key] = _cloneObject(object[key]);
+          if (object[key] instanceof jQuery) {
```

Comment:
Oh, my apologies for that, we always include jQuery on our pages so I didn't notice.